.github/workflows/go-test.yaml

```diff
@@ -47,7 +47,19 @@ jobs:
     - name: Test
       run: |
         # Exclude running unit tests against third_party repos.
-        go test -race ./...
+        go test -race -timeout=5m ./...
+
+    - run: |
+        cd "$(mktemp -d)"
+        wget -O protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v27.1/protoc-27.1-linux-x86_64.zip
```

Comment:
nit:
can you have the version match those of the chainguard sdk

```
We currently require protoc v5.28.0.

Install protoc codegen dependencies:

go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.34.2
go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.5.1
go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway@v2.22.0
go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2@v2.22.0
```

https://github.com/chainguard-dev/mono/blob/main/sdk/README.md

Reply:
`brew` pulled in 28.1, so I went with that (since downgrading is a PITA).

Reply:
If we could make it stop putting precise versions in the header comments, I'd wager we could make our lives much much easier.

Reply:
that would be nice